lib/binds.nix

```diff
@@ -67,6 +67,30 @@
       mkLuaBinding binding.value action binding.description;
 
     pushDownDefault = attr: mapAttrs (_: mkDefault) attr;
+
+    mkLznBinding = mode: key: action: desc: {
+      inherit mode desc key action;
+    };
+
+    mkLznExprBinding = mode: key: action: desc: {
+      inherit mode desc key action;
+      lua = true;
+      silent = true;
+      expr = true;
+    };
+
+    mkSetLznBinding = binding: action: {
+      inherit action;
+      key = binding.value;
+      desc = binding.description;
+    };
+
+    mkSetLuaLznBinding = binding: action: {
+      inherit action;
+      key = binding.value;
+      lua = true;
+      desc = binding.description;
+    };
```

Comment:
nah not too late, in fact now is probably the best time to get rid of it, I don't like how it needs to import the option file either, but I didn't bother touching it. in hindsight, it should be easy to do something similar by using `options ` instead of importing the file

any specific reason you don't like this

Reply:
Mainly because they are hard to trace/understand. I was specifically talking about these functions (and the ones for normal binds). Though the self imports might be more important to get rid of. What do you mean by `by using options instead`?

Reply:
too much pain, deferring this issue to the next unfortunate soul (I did change `self = import ...` to simply `options.foo` tho)